src/components/Button.js

```diff
@@ -233,6 +233,9 @@ class Button extends Component {
         return (
             <Pressable
                 onPress={(e) => {
+                    if (e.type === 'click') {
+                        e.currentTarget.blur();
+                    }
```

Comment:
Style - add another line break before the next if statement

Reply:
Done.